scripts/esbuild/testing.ts

```diff
@@ -0,0 +1,141 @@
+import type { BuildOptions as ESBuildOptions, Plugin } from 'esbuild';
+import fs from 'fs-extra';
+import { join } from 'path';
+
+import { copyTestingInternalDts } from '../bundles/testing';
+import { getBanner } from '../utils/banner';
+import type { BuildOptions } from '../utils/options';
+import { writePkgJson } from '../utils/write-pkg-json';
+import { getBaseEsbuildOptions, getEsbuildAliases, getEsbuildExternalModules, runBuilds } from './util';
+
+const EXTERNAL_TESTING_MODULES = [
```

Comment:
I think the built-in node modules should be marked as external when we set `platform: 'node'` (based [on the esbuild docs](https://esbuild.github.io/api/#platform)) so I think we should only need to put the npm packages in here

Reply:
or put another way, did you run into an issue where you needed to specifically mark things like `os` and `path` as external?

Reply:
I saw that a bunch have been already defined in [`externalNodeModules`](https://github.com/ionic-team/stencil/blob/2131f07bc30786669a34ac6dbef526d3cdfe0c86/scripts/esbuild/util.ts#L48) so will remove them from the list 👌